deployments/get/objects.go

```diff
@@ -283,6 +283,11 @@ func GenerateDaemonSet(env, namespace string) *appsv1.DaemonSet {
 			MountPath: "/media/root/etc/os-release",
 			ReadOnly:  true,
 		},
+		{
+			Name:      "procfs-path", //BPF (read-only)
+			MountPath: "/proc",
+			ReadOnly:  true,
+		},
```

Comment:
We will need to mount it in a custom directory, maybe /host/proc and configure KubeArmor to consider the custom procfs path wherever it's using it.

Reply:
@daemon1024 so i have to check if the env is k8s and if true i have to change "/proc" to "/host/proc" whereever the proc directory is used in the Kuberarmor